packages/ckeditor5-dev-tests/lib/utils/automated-tests/getwebpackconfig.js

```diff
@@ -37,14 +37,17 @@ module.exports = function getWebpackConfigForAutomatedTests( options ) {
 								}
 							}
 						},
+						'css-loader',
```

Comment:
Which library upgrade requires this?

Reply:
Answering my own question - that's because of postcss-loader v4 as mentioned here: https://github.com/ckeditor/ckeditor5/pull/8213/files#diff-dec446e3fa51a26c473acc17b1283060f6cc8ef8abed2752c70828e88a0bbb5bR219.

Reply:
See: https://github.com/webpack-contrib/postcss-loader/releases/tag/v4.0.0.

> loader output only CSS, so you need to use css-loader/file-loader/raw-loader to inject code inside bundle